docs/doxygen-user/multi-user/installSystems.dox

```diff
@@ -0,0 +1,42 @@
+/*! \page install_multiuser_systems_page Multi-user Hardware / VM Selection
+
+\section multiuser_system Pick Your Hardware / VM Configuration
+
+The first step in setting up a multi-user cluster is picking how many computers or VMs you'll use to run the various services. 
+
+From a service perspective, you'll need to run:
+- PostgeSQL database server
+- Apache Solr text indexing server
+- ActiveMQ messaging server
+- Network storage
+
+You can run each of these on their own dedicated VM, but that is not necessary.  We recommend:
+
+- Server 1: PostgreSQL and ActiveMQ 
+ - Both of these are low overhead services. 
+- Server 2: Apache Solr and Shared Storage (if you are using Windows File Sharing).
+ - Apache Solr uses a lot of memory, so we keep it by itself. 
+ - The benefit of running the shared storage on the same computer as Solr is performance. Solr is much faster when it can write to local storage instead of over the network.
+ - If you are using a different form of shared storage, then we still recommend keeping Solr on its own.  
```

Comment:
This section doesn't seem to flow very well. It might be better as a single bullet point? Like:
Apache Solr uses a lot of memory, so we recommend keeping it by itself. The exception is if you are using Windows File Sharing, since putting the shared storage on the same VM as Solr will increase performance. Solr is much faster when it can write to local storage instead of over the network. If you are using a different form of shared storage, then we recommend keeping Solr on its own.

Reply:
Added a line above the suggested configurations.